src/dependency-manager/src/interpolation/interpolation-context.ts

```diff
@@ -0,0 +1,6 @@
+import { ParameterValueV2 } from '../service-config/base';
+
+export interface InterpolationContext {
```

Comment:
Do we need this anymore? Shouldn't the context be a ServiceConfig?

Reply:
This was a duplicate file probably from a merge. Removed it. See answer regarding ServiceConfig below.